src/components/Input/Input.js

```diff
@@ -80,6 +80,7 @@ const inputBaseStyles = ({ theme }) => css`
   transition: border-color ${theme.transitions.default};
   width: 100%;
   ${textMega({ theme })};
+  margin: 0;
```

Comment:
This is a breaking change, isn't it?

Reply:
Nope, it's a fixing change :D

Reply:
Ah, I confused this with the margin on the outer container. All good 👍